src/autolabel/models/hf_pipeline.py

```diff
@@ -84,6 +97,49 @@ def __init__(self, config: AutolabelConfig, cache: BaseCache = None) -> None:
         # initialize LLM
         self.llm = HuggingFacePipeline(pipeline=pipe, model_kwargs=model_kwargs)
 
+    def _generate_sequence_bias(self) -> Dict:
+        """Generates sequence bias dict to add to the config for the labels specified
+
+        Returns:
+            Dict: sequence bias, max new tokens, and num beams
+        """
+        if len(self.config.labels_list()) == 0:
+            logger.warning(
+                "No labels specified in the config. Skipping logit bias generation."
+            )
+            return {}
+        try:
+            from transformers import AutoTokenizer
+        except ImportError:
+            raise ValueError(
+                "Could not import transformers python package. "
+                "Please it install it with `pip install transformers`."
+            )
+        tokenizer = AutoTokenizer.from_pretrained(
+            self.model_name, use_fast=False, add_prefix_space=True
+        )
+        sequence_bias = {
+            tuple([tokenizer.eos_token_id]): 100.0,
+            # tuple(tokenizer([label], add_special_tokens=False).input_ids[0]): float(
+            #     "inf"
+            # )
+            # for label in self.config.labels_list()
+        }
+        max_new_tokens = 0
+        for label in self.config.labels_list():
+            tokens = tuple(tokenizer([label], add_special_tokens=False).input_ids[0])
+            # sequence_bias[tokens] = 10.0
+            # sequence_bias[tuple([tokens[0]])] = 10.0
+            for token in tokens:
+                sequence_bias[tuple([token])] = 100.0
```

Comment:
Let's make "100" a number that is configurable? Instead of storing a boolean, config.logit_bias() can be set to a float. And if its set to 0 (which should be default) then we won't apply logit_bias.

Reply:
Sounds good to me